examples/svelte/imports/ui/App.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+  import { Meteor } from "meteor/meteor";
+  import { LinksCollection } from '../api/links';
+
+  /**
+   * Meteor tracker for Svelte
+   * {@link https://github.com/rdb/svelte-meteor-data}
+   */
+  import { useTracker } from 'meteor/rdb:svelte-meteor-data';
+
+  let counter = 0;
+  const addToCounter = () => {
+    counter += 1;
+  }
+
+  const links = useTracker(() => LinksCollection.find().fetch());
+</script>
+
+
+<div class="container">
+  <h1>Welcome to Meteor!</h1>
+
+  <button on:click={addToCounter}>Click Me</button>
+  <p>You've pressed the button {counter} times.</p>
+
+  <h2>Learn Meteor!</h2>
+  {#await Meteor.subscribe('todos')}
+    <ul>
+      {#each links as link (link._id)}
+        <li><a href={link.url} target="_blank" rel="noreferrer">{link.title}</a></li>
+      {/each}
+    </ul>
+  {:else}
+    <div>Loading ...</div>
+  {/if}
```

Comment:
The blocks must be swapped: `<div>Loading ...</div>` after `#await` and `<ul>` after `:else`.

Reply:
Couldn't really figure out the `{#await Meteor.subscribe('...')}` feature, Svelte complains about the usage of `:else`. 🤔 Can't figure out if that's my fault or there's something up with the client bundle. 😅

Reply:
Wright, I missed this one. The correct Svelte syntax is `{#await expression}...{:then}...{/await}`.  There is a mistake in [rdb:svelte-meteor-data README](https://github.com/rdb/svelte-meteor-data#subscriptions).